fixtures/dynamic-worker-loading/wrangler.jsonc

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "dynamic-worker-loading",
+	"main": "src/index.ts",
+	"compatibility_date": "2023-05-04",
+	"unsafe": {
+		"bindings": [
+			{
+				"name": "LOADER",
+				"type": "worker-loader",
+				"id": "random-id",
```

Comment:
What does `id` mean in this binding.
I guess my question is, "What does it mean to have multiple Worker Loaders?"

Reply:
Oh wait, no need to randomly generate. It can just be left null (not empty string, but null/omitted), in which case workerd will do the right thing.

Reply:
Will it be needed for production deployments? In other words can we just remove it from wrangler.jsonc altogether? Or just ignore it in local dev?

Reply:
Production does not support this option at all, that's why I recommend removing it.